File: reactapp4.client/src/interfaces/SportRadarMLBLeagueSchedule.ts
export interface SportRadarMLBLeagueSchedule {
  league_alias: string | null;
  league_name: string | null;
  league_id: string | null;              // uuid
  season_id: string | null;              // uuid
  season_year: number | null;
  season_type: string | null;
  game_id: string | null;                // uuid
  game_status: string | null;
  game_coverage: string | null;
  game_game_number: number | null;
  game_day_night: string | null;
  game_scheduled: string | null;         // ISO string with TZ
  game_home_team: string | null;         // uuid
  game_away_team: string | null;         // uuid
  game_attendance: number | null;
  game_duration: string | null;          // e.g. "2:10"
  game_double_header: boolean | null;
  game_entry_mode: string | null;
  game_reference: string | null;

  venue_name: string | null;
  venue_market: string | null;
  venue_capacity: number | null;
  venue_surface: string | null;
  venue_address: string | null;
  venue_city: string | null;
  venue_state: string | null;
  venue_zip: string | null;
  venue_country: string | null;
  venue_id: string | null;               // uuid
  venue_field_orientation: string | null;
  venue_stadium_type: string | null;
  venue_time_zone: string | null;
  venue_location_lat: number | null;
  venue_location_lng: number | null;

  home_name: string | null;
  home_market: string | null;
  home_abbr: string | null;
  home_id: string | null;                // uuid
  home_win: number | null;
  home_loss: number | null;

  away_name: string | null;
  away_market: string | null;
  away_abbr: string | null;
  away_id: string | null;                // uuid
  away_win: number | null;
  away_loss: number | null;

  broadcast_1_network: string | null;
  broadcast_1_type: string | null;
  broadcast_1_locale: string | null;
  broadcast_1_channel: string | null;
  broadcast_2_network: string | null;
  broadcast_2_type: string | null;
  broadcast_2_locale: string | null;
  broadcast_2_channel: string | null;
  broadcast_3_network: string | null;
  broadcast_3_type: string | null;
  broadcast_3_locale: string | null;
  broadcast_3_channel: string | null;

  game_rescheduled: string | null;       // ISO string with TZ (if present)
  game_parent_id: string | null;         // uuid
}

File: ReactApp4.Server/Data/SportRadarMLBLeagueSchedule.cs
using System;

namespace ReactApp4.Server.Data
{
    public class SportRadarMLBLeagueSchedule
    {
        public string? LeagueAlias { get; set; }
        public string? LeagueName { get; set; }
        public Guid? LeagueId { get; set; }
        public Guid? SeasonId { get; set; }
        public int? SeasonYear { get; set; }
        public string? SeasonType { get; set; }
        public Guid? GameId { get; set; }
        public string? GameStatus { get; set; }
        public string? GameCoverage { get; set; }
        public int? GameGameNumber { get; set; }
        public string? GameDayNight { get; set; }
        public DateTimeOffset? GameScheduled { get; set; }
        public Guid? GameHomeTeam { get; set; }
        public Guid? GameAwayTeam { get; set; }
        public double? GameAttendance { get; set; }
        public TimeSpan? GameDuration { get; set; }
        public bool? GameDoubleHeader { get; set; }
        public string? GameEntryMode { get; set; }
        public string? GameReference { get; set; }

        public string? VenueName { get; set; }
        public string? VenueMarket { get; set; }
        public int? VenueCapacity { get; set; }
        public string? VenueSurface { get; set; }
        public string? VenueAddress { get; set; }
        public string? VenueCity { get; set; }
        public string? VenueState { get; set; }
        public string? VenueZip { get; set; }
        public string? VenueCountry { get; set; }
        public Guid? VenueId { get; set; }
        public string? VenueFieldOrientation { get; set; }
        public string? VenueStadiumType { get; set; }
        public string? VenueTimeZone { get; set; }
        public double? VenueLocationLat { get; set; }
        public double? VenueLocationLng { get; set; }

        public string? HomeName { get; set; }
        public string? HomeMarket { get; set; }
        public string? HomeAbbr { get; set; }
        public Guid? HomeId { get; set; }
        public int? HomeWin { get; set; }
        public int? HomeLoss { get; set; }

        public string? AwayName { get; set; }
        public string? AwayMarket { get; set; }
        public string? AwayAbbr { get; set; }
        public Guid? AwayId { get; set; }
        public int? AwayWin { get; set; }
        public int? AwayLoss { get; set; }

        public string? Broadcast1Network { get; set; }
        public string? Broadcast1Type { get; set; }
        public string? Broadcast1Locale { get; set; }
        public string? Broadcast1Channel { get; set; }
        public string? Broadcast2Network { get; set; }
        public string? Broadcast2Type { get; set; }
        public string? Broadcast2Locale { get; set; }
        public string? Broadcast2Channel { get; set; }
        public string? Broadcast3Network { get; set; }
        public string? Broadcast3Type { get; set; }
        public string? Broadcast3Locale { get; set; }
        public string? Broadcast3Channel { get; set; }
        public string? GameRescheduled { get; set; }
        public Guid? GameParentId { get; set; }
    }
}
